README.md

```diff
@@ -1,50 +1,52 @@
-# Raiden Network
+# Raiden
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/brainbot-com/raiden?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/raiden-network/raiden?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-Raiden Network is a proposed extension to Ethereum which scales-out asset transfer capacity.
+Raiden is a proposed extension to Ethereum which scales-out asset transfer capacity in the network. It is inspired by [Lightning Network](https://lightning.network/) which leverages off-chain asset transfers to carry out the vast majority of transactions. For more information please visit http://raiden.network/.

```

Comment:
Yes I actually thought about that for a while, because it seems to be used interchangeably. In the end I chose to use it the way the README originally  referred to Raiden Network. I'll change it back.